is/tuple_of.ts

```diff
@@ -93,6 +125,23 @@ export function isTupleOf<
       predTup,
       predRest,
     );
+  } else {
+    return rewriteName(
+      (x: unknown): x is [...TupleOf<T>, ...PredicateType<R>] => {
```

Comment:
Maybe `x is [...TupleOf<T>, ...PredicateType<R>, ...TupleOf<T>]` right?

Reply:
`x is [...TupleOf<T>, ...PredicateType<R>, ...TupleOf<L>]` is right.
The last type parameter is `L`.
And I fixed overloads too.